tools/collect-mod-repro-bundle.ps1

```
@@ -157,24 +187,37 @@ function Get-LaunchContext {
             reasonCode = "python_not_found"
             confidence = 0.0
             launchKind = "Unknown"
+            source = $forcedSource
+            forcedWorkshopIds = $normalizedForcedWorkshopIds
+            forcedProfileId = $normalizedForcedProfileId
         }
     }
 
-    $pythonArgs = @()
+    $args = @()
```

PSScriptAnalyzer (1.24.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)